interface/patient_file/report/custom_report.php

```diff
@@ -638,7 +637,7 @@ function postToGet($arin)
                         unlink($from_file_tmp_name);
 
                         // Make sure whatever follows is on a new page.
-                        // $pdf->AddPage(); // Only needed for signature line. Patched out 04/20/2017 sjpadgett.
```

Comment:
hi @sjpadgett ,
Had to add this back to fix a bug. If the report has the following in this order:
1. PDF
2. Image (jpg)
3. PDF
Then the image gets superimposed on the following PDF

Reply:
I think I patched out because it mostly would add an additional blank page. Not completely sure that was the case because it was more than a day ago:) but sounds like something i'd do.
No biggy.
Also note that if any of these updates get back ported to 501 one would need to patch out the use mPDF. One last thing, i'm not sure if mpdf supports barcode and I believe tcpdf is used for this but not sure if html2pdf is anywhere.